### ISSUE TITLE
FR: give project creation rights to user admin accounts by default

### ISSUE THREAD

**Issue by keunes** [Project-Pier/ProjectPier-Core#44]
After initial installation, I deleted the automatically created 'Welcome' project. Then tried to **create a new project**, but got the following message:

'You don't have permission to access the requested page'

Also can't **edit permissions** when clicking on my user account name in the top right menu (leads to same error message).

**Comment by keunes** [comment]
Allright. Found out how to get it working: http://www.projectpier.org/node/3905#comment-9749

Also just [read](http://www.projectpier.org/node/3030#comment-8063) that this behaviour is intended:

> The previous versions allowed any administrator to create a project. The requirement from the community was to be able to allow any user to create projects. So, now administrators are not allowed to create projects by default. Only users that have this option in there profile can create projects.

I would propose to change default behaviour so that the _first user (i.e. Administrator)_ does have the rights to create a new project. I don't see any objections to combining the following two:
- "requirement from the community was to be able to allow any user to create projects"
- allow administrator user accounts to by default have the right to create projects

This would greatly enhance usability for new users.

EDIT: Change issue title accordingly.

**Comment by Engelbert** [comment]
:+1: 
I second this.

**Comment by UsrCletus** [comment]
The link where you state how to correct this has been taken down. Can I get an assist?

**Comment by federico-razzoli** [comment]
The link does not work anymore. How did you solve the problem?

**Comment by keunes** [comment]
@federico-razzoli I stopped looking into project pier long time ago. Seems the whole project is abandoned (last commit was on 21 Aug 2015, and whole project website is down).

**Comment by federico-razzoli** [comment]
@keunes you are right. Though, I couldn't find anything better in the open source space. I'm considering to fork it.

**Comment by guiperry** [comment]
> @keunes you are right. Though, I couldn't find anything better in the open-source space. I'm considering to fork it.

You should. I'd be glad to assist in bringing this project back to life as well! The only other open-source option there is for project management platforms is redmine, and that is hell to work with. This is PHP, so it shouldn't be as complex as the Ruby on Rails implementation in redmine.

**Comment by federico-razzoli** [comment]
@cybereq If you want I'll keep you in mind and will contact you with GitHub private messages.

**Comment by pushpen** [comment]
has anybody forked this?
